drivers/hmis/app/models/hmis/hud/processors/current_living_situation_processor.rb

```diff
@@ -6,6 +6,37 @@
 
 module Hmis::Hud::Processors
   class CurrentLivingSituationProcessor < Base
+    def process(field, value)
+      attribute_name = ar_attribute_name(field)
+      current_living_situation = @processor.send(factory_name)
+
+      return super(field, value) unless attribute_name == 'verified_by_project_id'
+
+      verified_by_project_id = attribute_value_for_enum(graphql_enum(field), value)
```

Comment:
I was gonna say that we don't need `attribute_value_for_enum` since this isn't an enum field, but, I think we do need it because it converts `_HIDDEN`=>`nil`. We could skip the `graphql_enum` bit since we know that this isn't an enum field. Looks like we have this exact situation here: https://github.com/greenriver/hmis-warehouse/blob/aa9a48f3c1fc2ed68a7738a17ba4b119d9f7ec02/drivers/hmis/app/models/hmis/hud/processors/base.rb#L230

This points towards the need for a refactor, but for now maybe removing the `graphql_enum` and adding a comment is sufficient

Reply:
I was suggesting to just do what the linked line of code does - pass `nil` for the first argument. But yeah that sounds better, if you can add another helper method for the hidden change. Thanks!